immich-app/immich
# feat(web): use time buckets of person detail page (3)

In this PR:
- Use time buckets on person detail page
- Use time buckets on feature face selection page
- Add an option to `AssetGrid` to allow for single selection (fires an on:select with the asset and doesn't show the multi-day/month selection options)
- Move fullscreen modal + outclick to the SuggestionModel component and it maps that to the cancel event
- Auto-scroll to top after a single selection (in AssetGrid)
- Fix an issue where canceling the suggest merge modal would stop editing the name input
- Add shift select to person detail view (#3550)
- View all assets for a person (#2472)

Tested Scenarios:
- Edit a name to a duplicate, click yes/no/close/out-click
- Edit a name to a new name, saved
- Select another feature photo, updated
- Cancel feature photo selection, back to asset grid
- Open merge modal via menu
- Cancel merge modal
- Confirm merge a person in merge modal
-
